internal/persistence/queries_test.go

```diff
@@ -0,0 +1,359 @@
+		_, err = InsertManualTL(testDB, taskID, tlBeginTS, tlEndTS, comment)
+		require.NoError(t, err, "failed to insert task log")
+
+		// WHEN
+		entries, err := FetchStats(testDB, 100)
+
+		// THEN
+		require.NoError(t, err, "failed to fetch report entries")
+		require.Len(t, entries, 2)
+
+		assert.Equal(t, 1, entries[0].TaskID)
+		assert.Equal(t, 3, entries[0].NumEntries)
+		assert.Equal(t, 5*secsInOneHour+numSeconds, entries[0].SecsSpent)
+
+		assert.Equal(t, 2, entries[1].TaskID)
+		assert.Equal(t, 1, entries[1].NumEntries)
+		assert.Equal(t, 4*secsInOneHour, entries[1].SecsSpent)
+	})
+
+	t.Run("TestFetchStatsBetweenTS", func(t *testing.T) {
+		t.Cleanup(func() { cleanupDB(t, testDB) })
+
+		// GIVEN
+		referenceTS := time.Date(2024, time.September, 1, 9, 0, 0, 0, time.Local)
+		seedData := getTestData(referenceTS)
+		seedDB(t, testDB, seedData)
+
+		taskID := 1
+		numSeconds := 60 * 90
+		tlEndTS := referenceTS.Add(time.Hour * 2)
+		tlBeginTS := tlEndTS.Add(time.Second * -1 * time.Duration(numSeconds))
+		_, err = InsertManualTL(testDB, taskID, tlBeginTS, tlEndTS, taskLogComment)
+		require.NoError(t, err, "failed to insert task log")
+
+		// WHEN
+		reportBeginTS := referenceTS.Add(time.Hour * 24 * 7 * -2)
+		entries, err := FetchStatsBetweenTS(testDB, reportBeginTS, referenceTS, 100)
+
+		// THEN
+		require.NoError(t, err, "failed to fetch report entries")
+		require.Len(t, entries, 2)
+
+		assert.Equal(t, 1, entries[0].TaskID)
+		assert.Equal(t, 2, entries[0].NumEntries)
+		assert.Equal(t, 5*secsInOneHour, entries[0].SecsSpent)
+
+		assert.Equal(t, 2, entries[1].TaskID)
+		assert.Equal(t, 1, entries[1].NumEntries)
+		assert.Equal(t, 4*secsInOneHour, entries[1].SecsSpent)
+	})
+
+	t.Run("TestFetchReportBetweenTS", func(t *testing.T) {
+		t.Cleanup(func() { cleanupDB(t, testDB) })
+
+		// GIVEN
+		referenceTS := time.Date(2024, time.September, 1, 9, 0, 0, 0, time.Local)
+		seedData := getTestData(referenceTS)
+		seedDB(t, testDB, seedData)
+
+		taskID := 1
+		numSeconds := 60 * 90
+		tlEndTS := referenceTS.Add(time.Hour * 2)
+		tlBeginTS := tlEndTS.Add(time.Second * -1 * time.Duration(numSeconds))
+		_, err = InsertManualTL(testDB, taskID, tlBeginTS, tlEndTS, taskLogComment)
+		require.NoError(t, err, "failed to insert task log")
+
+		// WHEN
+		reportBeginTS := referenceTS.Add(time.Hour * 24 * 7 * -2)
+		entries, err := FetchReportBetweenTS(testDB, reportBeginTS, referenceTS, 100)
+
+		// THEN
+		require.NoError(t, err, "failed to fetch report entries")
+
+		require.Len(t, entries, 2)
+		assert.Equal(t, 2, entries[0].TaskID)
+		assert.Equal(t, 1, entries[0].NumEntries)
+		assert.Equal(t, 4*secsInOneHour, entries[0].SecsSpent)
+
+		assert.Equal(t, 1, entries[1].TaskID)
+		assert.Equal(t, 2, entries[1].NumEntries)
+		assert.Equal(t, 5*secsInOneHour, entries[1].SecsSpent)
+	})
+
+	err = testDB.Close()
+	require.NoErrorf(t, err, "error closing DB: %v", err)
+}
+
+func cleanupDB(t *testing.T, testDB *sql.DB) {
+	var err error
```

Comment:
```suggestion
func cleanupDB(t *testing.T, testDB *sql.DB) {
	t.Helper()

	var err error
```

You van enable thelper linter in golangci-lint